### DISCUSSION TITLE
建议增加maa的全域定时数量

### DISCUSSION THREAD

**Discussion by IAMTERRIFIED** [MaaAssistantArknights/MaaAssistantArknights#9661] [Ideas]
大号小号加上朋友的号感觉一天八个定时不够用，而且只能一天两清，想要三清的话还得再开一个maa和模拟器，有点占内存，大佬们能不能开发，可不可以自定义添加定时数量或者提升一下数量上限，非常感谢！

**Comment by Constrat** [comment]
Can't you use multiple MAAs with the same config on the same emulator?
For the generic user I think 8 is even more than enough.
As a simple rule I would give one MAA per account only.
